### PR TITLE
2021 04 01 adelacecilia-html coding session number five

### PR DESCRIPTION
**Pull request by adelacecilia**

### CODE REVIEW

File: web/adelacecilia/html/colgrup tag/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      th,
+      td {
+        border: 3px solid #0d4686;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>The colgroup tag</h1>
+
+    <table>
```
Comment (QuintyHH) [inline web/adelacecilia/html/colgrup tag/index.html:14]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/data list/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <body>
+    <label for="ice-cream-choice">
+      <h2> What's your favorite ice-cream? </h2>
+      <h3> Please select an option: </h3>
+    <input list="ice-cream-flavors"
+    id="ice-cream-choice"
+    name="ice-cream-choice" 
+    </label>
+
+    <datalist id="ice-cream-flavors">
```
Comment (QuintyHH) [inline web/adelacecilia/html/data list/index.html:13]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/datalist tag/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>The "datalist" tag</h1>
```
Comment (QuintyHH) [inline web/adelacecilia/html/datalist tag/index.html]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/div tag/index.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  border: 7px outset #20d120;
+  background-color: black;    
+  text-align: center;
+  color: #20d120;
+}
+</style>
+</head>
+<body>
+
+<h1>The "div" element</h1>
```
Comment (QuintyHH) [inline web/adelacecilia/html/div tag/index.html]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/embed tag/index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>An embedded HTML page</title>
+  </head>
+  <body>
+    <h1>Embedded HTML page</h1>
+
+    <embed
```
Comment (QuintyHH) [inline web/adelacecilia/html/embed tag/index.html:9]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/footer/index.html [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <title>Footer</title>
+<body>
+
+<h1>The "footer" tag</h1>
```
Comment (QuintyHH) [inline web/adelacecilia/html/footer/index.html]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/input type/index.html [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1>Telephone Input Field</h1>
+
+    <form action="/action_page.php">
```
Comment (QuintyHH) [inline web/adelacecilia/html/input type/index.html:6]: Please dont leave empty lines in your HTML :)

File: web/adelacecilia/html/selectors/index.html [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    <div class="selector-universal">
+      <p>My favorite three books</p>
+      <ul>
+        <li>The Alchemist by Paulo Coelho</li>
+        <li>Honest Illusions by Nora Roberts</li>
+        <li>Me Before You by Jojo Moyers</li>
+      </ul>
+    </div>
+
+    <h2>#X Selector</h2>
```
Comment (QuintyHH) [inline web/adelacecilia/html/selectors/index.html:16]: Please dont leave empty lines in your HTML :)

**Review by QuintyHH — CHANGES_REQUESTED** [review]
Please dont leave empty lines in your HTML :)

**Review by psyhedeliq — APPROVED** [review]

File: web/adelacecilia/html/datalist tag/index.html (hunk 3 above)
Comment (vasilemaftita) [reply web/adelacecilia/html/datalist tag/index.html]: please delete blank line 4.

File: web/adelacecilia/html/div tag/index.html (hunk 4 above)
Comment (vasilemaftita) [reply web/adelacecilia/html/div tag/index.html]: please delete blank line 15

File: web/adelacecilia/html/embed tag/index.html (hunk 5 above)
Comment (vasilemaftita) [reply web/adelacecilia/html/embed tag/index.html:9]: please delete blank line 8

File: web/adelacecilia/html/footer/index.html (hunk 6 above)
Comment (vasilemaftita) [reply web/adelacecilia/html/footer/index.html]: please delete blank line 5

**Review by vasilemaftita — APPROVED** [review]
please delete blank line { .... } va rog stergeti spatiile, liniile goale.

**Review by RalucaRbc — APPROVED** [review]
Apart from those empty lines in the .html files everything looks fine I think 👍

**Review by biggicode — APPROVED** [review]

File: web/adelacecilia/html/data list/index.html (hunk 2 above)
Comment (adelacecilia) [reply web/adelacecilia/html/data list/index.html:13]: oops...I did it again.  :[

Sorted!

File: web/adelacecilia/html/datalist tag/index.html (hunk 3 above)
Comment (adelacecilia) [reply web/adelacecilia/html/datalist tag/index.html]: Sorted! 

Thank you!

**Review by QuintyHH — APPROVED** [review]
:100:

**Review by swoopstyle — APPROVED** [review]
💯

**Review by robertoalx — APPROVED** [review]
💯

**Pull request merged**